horiz_interp/include/horiz_interp_conserve.inc

```diff
@@ -296,15 +189,15 @@ contains
 
     do j = 1,nlat_in
        do i = 1,nlon_in
-          Interp%area_src(i,j) = dlon_in(i) * dsph_in(j)
+          Interp%HI_KIND_TYPE_%area_src(i,j) = dlon_in(i) * dsph_in(j)
```

Comment:
should this math be done in r8?

Reply:
I don't think so, the other values aren't r8

Reply:
@bensonr